extensions/geode-modules/src/main/java/org/apache/geode/modules/util/BootstrappingFunction.java

```diff
@@ -97,6 +100,9 @@ protected Cache verifyCacheExists() {
     }
 
     if (cache == null) {
+      if (logger.isDebugEnabled()) {
+        logger.debug("Creating a new cache in BootstrappingFunction");
```

Comment:
Is this the proper log level?

Reply:
I'd suggest to setting it to `info` so that it's always visible.

Reply:
Good idea!